src/Symfony/Component/HttpClient/DataCollector/HttpClientDataCollector.php

```diff
@@ -36,6 +36,7 @@ final class HttpClientDataCollector extends DataCollector implements LateDataCol
     public function registerClient(string $name, TraceableHttpClient $client): void
     {
         $this->clients[$name] = $client;
+        $client->enabled(true);
```

Comment:
I'd rather have `setEnabled(true)` that having a method that is a mix of a getter and a setter